doc/source/api/series.rst

```diff
@@ -278,14 +278,34 @@ Time series-related
    Series.tshift
    Series.slice_shift
 
+Accessors
+---------
+
+Pandas provides dtype-specific methods under various accessors.
+These are separate namespaces within :class:`Series` that only apply
+to specific data types.
+
+=========================== =================================
+Data Type                   Accessor
+=========================== =================================
+Datetime, Timedelta, Period :ref:`dt <api.series.dt>`
+String                      :ref:`str <api.series.str>`
+Categorical                 :ref:`cat <api.series.cat>`
+Sparse                      :ref:`sparse <api.series.sparse>`
+=========================== =================================
+
+.. _api.series.dt:
+
 Datetimelike Properties
------------------------
+~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
These were all moved down one level. The TOD is now

```
Accessors
    - dt
    - str
    - cat
    - sparse
```

Rather than having them all at the top-level of this section.